python/interpret_community/community/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 """Module for model interpretability, including feature and class importance for blackbox and greybox models."""
 
-from .tabular_explainer import TabularExplainer
+from interpret_community import TabularExplainer
```

Comment:
I was trying to remove the sphinx error (shown below) and making this change removed the error. Is this correct? Or should I revert?

**Sphinx error:**
C:\Users\xxx\Documents\GitHub\interpret-community\python\docbuild\interpret_community.community.rst:7: WARNING: autodoc: failed to import module 'interpret_community.community'; the following exception was raised:
Traceback (most recent call last):
  File "c:\users\xxx\appdata\local\programs\python\python37-32\lib\site-packages\sphinx\ext\autodoc.py", line 551, in import_object
    __import__(self.modname)
  File "C:\Users\xxx\Documents\GitHub\interpret-community\python\interpret_community\community\__init__.py", line 7, in <module>
    from .tabular_explainer import TabularExplainer
ModuleNotFoundError: No module named 'interpret_community.community.tabular_explainer'

Reply:
from interpret_community.tabulat_explainer import TabularExplainer

Reply:
Thanks. Changed.